pkg/frontend/adminactions/kubeactions.go

```diff
@@ -122,11 +122,16 @@ func (k *kubeActions) KubeCreateOrUpdate(ctx context.Context, o *unstructured.Un
 	return err
 }
 
-func (k *kubeActions) KubeDelete(ctx context.Context, groupKind, namespace, name string) error {
+func (k *kubeActions) KubeDelete(ctx context.Context, groupKind, namespace, name string, force bool) error {
 	gvr, err := k.gvrResolver.Resolve(groupKind, "")
 	if err != nil {
 		return err
 	}
 
-	return k.dyn.Resource(*gvr).Namespace(namespace).Delete(ctx, name, metav1.DeleteOptions{})
+	resourceDeleteOptions := metav1.DeleteOptions{}
+	if force {
+		resourceDeleteOptions.GracePeriodSeconds = func(i int64) *int64 { return &i }(0)
```

Comment:
There is a helper library provided by autorest: `"github.com/Azure/go-autorest/autorest/to"` you can use its convert

```
resourceDeleteOptions.GracePeriodSeconds = to.Int64Ptr(0)
```
it will be consistent with the rest of the codebase. Thank you.

Reply:
Perfect! Line has been modified to be consistent with the rest of the codebase. Thank you for pointing that out.